README.md

```diff
@@ -108,8 +108,10 @@ PCFParameters:
 # Collect additional metrics? If false, all settings in this section below are ignored. 
 # If yes, additional metrics will be collected for all queues, channels and listeners listed below. 
   sendPCFCommands: true
-# Use wildcards? If yes, only one PCF command will be send, matching all objects on queue manager. Otherwise, each 
-# object will be monitored by separate PCF command.  
+# If usePCFWildcards equals "true", then all monitored objects will be grouped by object type: QUEUE, CHANNEL and LISTENER. Only one PCF command will be sent for each object type. 
+# If usePCFWildcards equals "false", then PCF command will be send for each object.
+# Each PCF command uses a separate connection, so sending a large number of PCF commands will create a large number of connections to MQ queue manager.
+# For example, for 100 monitoring queues will be opened 100 connections to MQ queue manager, if usePCFwildcard equals "false". 
```

Comment:
Both pros and cons have to be described.
Using wildcards has it's disadvantages too: if we want to monitor a couple of queues while having 10.000 queues in the QM and usePCFwildcard is set to "true", we will be facing performance issues.
The fact has to be mentioned. But use more "official" language.